cognee/infrastructure/databases/graph/neptune_driver/adapter.py

```diff
@@ -997,6 +1080,298 @@ async def get_connections(self, node_id: UUID) -> list:
             raise Exception(f"Failed to get connections: {error_msg}")
 
 
+    async def remove_connection_to_predecessors_of(
+        self, node_ids: list[str], edge_label: str
+    ) -> None:
+        """
+        Remove connections (edges) to all predecessors of specified nodes based on edge label.
+
+        Parameters:
+        -----------
+
+            - node_ids (list[str]): A list of IDs of nodes from which connections are to be
+              removed.
+            - edge_label (str): The label of the edges to remove.
+
+        Returns:
+        --------
+
+            - None: None
+        """
+        query = f"""
+        UNWIND $node_ids AS node_id
+        MATCH ({{`~id`: node_id}})-[r:{edge_label}]->(predecessor)
+        DELETE r;
+        """
+        params = {"node_ids": node_ids}
+        await self.query(query, params)
+
+
+    async def remove_connection_to_successors_of(
+        self, node_ids: list[str], edge_label: str
+    ) -> None:
+        """
+        Remove connections (edges) to all successors of specified nodes based on edge label.
+
+        Parameters:
+        -----------
+
+            - node_ids (list[str]): A list of IDs of nodes from which connections are to be
+              removed.
+            - edge_label (str): The label of the edges to remove.
+
+        Returns:
+        --------
+
+            - None: None
+        """
+        query = f"""
+        UNWIND $node_ids AS node_id
+        MATCH ({{`~id`: node_id}})<-[r:{edge_label}]-(successor)
+        DELETE r;
+        """
+        params = {"node_ids": node_ids}
+        await self.query(query, params)
+
+
+    async def get_node_labels_string(self):
+        """
+        Fetch all node labels from the database and return them as a formatted string.
+
+        Returns:
+        --------
+
+            A formatted string of node labels.
+        """
+        node_labels_query = "CALL neptune.graph.pg_schema() YIELD schema RETURN schema.nodeLabels as labels "
+        node_labels_result = await self.query(node_labels_query)
+        node_labels = node_labels_result[0]["labels"] if node_labels_result else []
+
+        if not node_labels:
+            raise ValueError("No node labels found in the database")
```

Comment:
I'm assuming this is what its doing in other adapters... 
Please add this to the documentation under "Raises"